ert_storage/app.py

```diff
@@ -36,6 +36,16 @@ def render(self, content: Any) -> bytes:
 )
 
```

Comment:
Not related to your change, but there is a lot of unused imports in this file  - possible to do a cleanup? 😬